test.sh

```diff
@@ -10,6 +10,7 @@
 #      2: normal 32bit tests
 #      3: tsi build
 #      count: print the number of test environments
+#      flux: run Flux e2e tests via the external test harness
```

Comment:
I didn't add a new number for this because I wanted these tests to run in a separate CI job

Reply:
I like this - should we plan to move the rest of the tests to their own CI jobs?

Reply:
I would vote for yes (2.x is already there, it's nice to see more fine-grained reporting)

Reply:
@codyshepherd FYI ☝️

Reply:
@lesam would you say that the following issues capture the to-do indicated by this conversation? Or do we need a new issue?

influxdata/influxdb#17573
influxdata/influxdb#21074
influxdata/plutonium#3103